script/build.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
```

Comment:
Just wondering if we could simply put `ember build` in the `package.json`? The same question goes for `ember test`. And did `-eu` flags have any effect? If yes, was it reflected in your scripts?

Reply:
About only calling ember, it's a good question! I'll test later!
About the flags, it just guarantee that script would finalize at first error and variables should be declared before usage! I believe that won't interfere in the ember execution!